CHANGELOG/CHANGELOG-2.5.md

```diff
@@ -0,0 +1,80 @@
+- Accelerate Pod-to-Pod networking in `noEncap` and `hybrid` modes by leveraging nftables `flowtable` to reduce host network stack overhead. ([#7324](https://github.com/antrea-io/antrea/pull/7324), [@hongliangl])
+
+### Changed
+
+- Upgrade Go to 1.25. ([#7561](https://github.com/antrea-io/antrea/pull/7561), [@antoninbas])
+- Deprecate `L7FlowExporter` feature. ([#7567](https://github.com/antrea-io/antrea/pull/7567), [@luolanzone])
+- Increase the minimum supported Kubernetes version to 1.23. ([#7564](https://github.com/antrea-io/antrea/pull/7564), [@antoninbas])
+- Add `uniqueMACForSubInterfaces` option to Egress, enabling unique MAC addresses for Egress VLAN interfaces by default. ([#7599](https://github.com/antrea-io/antrea/pull/7599), [@luolanzone])
+- Evolve AntreaProxy with a new healthz server on port 10256, and with `PreferSameTrafficDistribution` support. ([#7371](https://github.com/antrea-io/antrea/pull/7371), [@hongliangl])
+- Update `FlowExporterDestination` to allow specifying the namespace and name for TLS certificate resources, supporting cross-namespace configurations. ([#7549](https://github.com/antrea-io/antrea/pull/7549), [@andrew-su])
+- Promote feature gates `TopologyAwareHints` and `ServiceTrafficDistribution` to General Availability (GA). ([#7503](https://github.com/antrea-io/antrea/pull/7503), [@hongliangl])
+- Improve FlowExporter performance by using netlink zone filtering when dumping conntrack flows, reducing CPU and memory usage. ([#7504](https://github.com/antrea-io/antrea/pull/7504), [@antoninbas])
+- Prevent WireGuard enablement when the traffic mode is not Encap. ([#7464](https://github.com/antrea-io/antrea/pull/7464), [@luolanzone])
+- Add disclaimers to Azure-related documentation and code, as Antrea is no longer tested on Azure. ([#7381](https://github.com/antrea-io/antrea/pull/7381), [@edwardbadboy])
+- Migrate dependency update configuration from Dependabot to Renovate to unify dependency management. ([#7354](https://github.com/antrea-io/antrea/pull/7354), [@ghadeer-elsalhawy])
+- Remove `AntreaProxy` and `NodePortLocal` feature gates from Helm charts and standard manifests. ([#7505](https://github.com/antrea-io/antrea/pull/7505), [@hongliangl])
+- Add validation to ensure `AntreaIPAM` is enabled with `SecondaryNetwork`. ([#7556](https://github.com/antrea-io/antrea/pull/7556), [@luolanzone])
+- Periodically sync ip rules to ensure consistency. ([#7295](https://github.com/antrea-io/antrea/pull/7295), [@hongliangl])
+- Use a more robust way to extract the source Node IP from encapsulated IGMP messages for multicast when the traffic mode is Encap. ([#7282](https://github.com/antrea-io/antrea/pull/7282), [@hongliangl])
+- Add validation to clarify `networkPolicyOnly` mode is not supported with Antrea Multicast. ([#7362](https://github.com/antrea-io/antrea/pull/7362), [@wenyingd])
+
+### Fixed
+
+- Upgrade CNI plugins to v1.8.0 to fix CVEs. ([#7397](https://github.com/antrea-io/antrea/pull/7397), [@luolanzone])
```

Comment:
should be omitted or lower priority, as I doubt the CVEs could be exploited in the context of Antrea